sdk/appconfiguration/spring-cloud-azure-feature-management-web/src/main/java/com/azure/spring/cloud/feature/manager/DynamicFeatureManagerSnapshot.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.feature.manager;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import reactor.core.publisher.Mono;
+
+/**
+ * Holds information on Feature Management properties and can check if a given
+ * feature is enabled. Returns the same value in the same request.
+ */
+public class DynamicFeatureManagerSnapshot {
```

Comment:
They will be.  They are the same thing as the non-snapshot version. The idea is that with the snapshot version each request is only evaluated once per request scope. Otherwise calling DynamicFeatureManager twice evaluates it completely twice, where this just returns what it did the last time.